src/installing-workflow/system-requirements.md

```diff
@@ -26,12 +26,9 @@ application footprint as well.
 Running smaller machines will likely result in increased system load and has been known to result in component failures
 and instability.
 
-## Docker Insecure Registry
-
-The on-cluster Docker registry is not deployed with TLS enabled. As such, all Kubernetes worker nodes must have their
-Docker daemons configured to use an **insecure registry**. The configured subnet should encompass any private networks
-used by your worker nodes, including overlay networks.
+*** WARNING ***
```

Comment:
I'd rather keep the old documentation, like this:

```
## Docker Insecure Registry

The on-cluster Docker registry is not deployed with TLS enabled. As such, versions prior to v2.2.0 must have all Docker daemons on the Kubernetes worker nodes configured to use an **insecure registry**. The configured subnet should encompass any private networks used by your worker nodes, including overlay networks.
```

In the future, if you want to add a warning in the docs, use the [admonition syntax](https://www.google.ca/#q=admonition+syntax+mkdocs&gws_rd=cr) so we can come up with a consistent style for these warnings.

```
!!! warning

    Versions prior to v2.2 require '--insecure-registry` to function properly.
```